src/client/connstring.rs

```diff
@@ -0,0 +1,311 @@
+use std::ascii::AsciiExt;
+use std::collections::HashMap;
+use std::vec::Vec;
```

Comment:
This is [in the prelude](http://doc.rust-lang.org/stable/std/prelude/), and so does not need to be explicitly imported.